res/values/strings.xml

```diff
@@ -199,7 +199,7 @@
     <string name="menu_add_attachment">Add Attachment</string>
     <string name="menu_leave_group">Leave Group</string>
     <string name="menu_delete_chat">Delete Chat</string>
-    <string name="ask_delete_named_chat">Are you sure you want to delete \"%1$s\"?</string>
+    <string name="ask_delete_named_chat">Are you sure you want to delete chat with %1$s?</string>
```

Comment:
this wording does not match when it is a groupname. why remove the quotes?

Reply:
Whether the variable here represents a group or individual, the deletion will only occur for one person. The person performing the deletion is chatting with the group or the person.

Reply:
i do not think, the addition makes it clear that the chat is not deleted on other people's devices (i am also not sure if this additional information is needed). but make things worse for groups.

as said,  in general, we prefer rewordings only if there is concrete user feedback (by quite some users!) or if there is something reworked in that area.

Reply:
To make it clear whose copy is being deleted, I would suggest "delete your copy of *". This line could be an isolated PR as it might be useful to have it's own discussion.